Sources/PointFree/Account/Account.swift

```diff
@@ -11,13 +11,310 @@ import Tuple
+      ]),
+    gridColumn(sizes: [:], [`class`([Class.grid.end(.mobile)])], [
+      form([action(path(to: .invite(.resend(invite.id)))), method(.post), `class`([Class.display.inlineBlock])], [
+        input([type(.submit), `class`([Class.pf.components.button(color: .purple, size: .small)]), value("Resend")])
+        ]),
+
+      form([action(path(to: .invite(.revoke(invite.id)))), method(.post), `class`([Class.display.inlineBlock, Class.padding([.mobile: [.left: 2]])])], [
+        input([type(.submit), `class`([Class.pf.components.button(color: .purple, size: .small)]), value("Revoke")])
+        ]),
+      ]),
+    ])
+}
+
+private let subscriptionInviteMoreRowView = View<Prelude.Unit> { _ in
+  gridRow([`class`([subscriptionInfoRowClass])], [
+    gridColumn(sizes: [.mobile: 2], [
+      div([
+        p(["Invite more"])
+        ])
+      ]),
+    gridColumn(sizes: [.mobile: 10], [
+      div([`class`([Class.padding([.mobile: [.leftRight: 1]])])], [
+        p(["You have 10 open spots on your team. Invite a team member below:"]),
+
+        form([action(path(to: .invite(.send(nil)))), method(.post)], [
+          input([
+            type(.email),
+            `class`([smallInputClass, Class.align.middle]),
+            name("email")]),
+
+          input([
+            type(.submit),
+            `class`([
+              Class.pf.components.button(color: .purple, size: .small),
+              Class.align.middle,
+              Class.margin([.mobile: [.left: 1]])
+              ]),
+            value("Add team member")])
+          ])
+        ])
+      ])
+    ])
+}
+
+private let subscriptionPaymentInfoView = View<Prelude.Unit> { _ in
+  gridRow([`class`([subscriptionInfoRowClass])], [
+    gridColumn(sizes: [.mobile: 3, .desktop: 2], [
+      div([
+        p(["Payment"])
+        ])
+      ]),
+    gridColumn(sizes: [.mobile: 9, .desktop: 5], [
+      div([`class`([Class.padding([.mobile: [.leftRight: 1]])])], [
+        p(["American Express 3*** ****** *2002"]),
+        p(["Expiration: 1/2018"]),
+        p(["Next payment due: 2017-12-24"]),
+        p(["Total Amount: $18.00"])
+        ])
+      ]),
+    gridColumn(sizes: [.mobile: 12, .desktop: 5], [
+      div([`class`([Class.grid.end(.mobile)])], [
+        p([
+          a([href("#"), `class`([Class.pf.components.button(color: .purple, size: .small)])], ["Update payment method"])
+          ])
+        ])
+      ])
+    ])
+}
+
+private let logoutView = View<Prelude.Unit> { _ in
+  gridRow([
+    gridColumn(sizes: [.mobile: 12], [
+      a([`class`([Class.pf.components.button(color: .black)]), href(path(to: .logout))], ["Logout"])
+      ])
+    ])
+}
+
+private let subscriptionInfoRowClass =
+  Class.border.bottom
+    | Class.pf.colors.border.gray800
+    | Class.padding([.mobile: [.topBottom: 3]])
+
+private let labelClass =
+  Class.h5
+    | Class.type.bold
+    | Class.display.block
+    | Class.margin([.mobile: [.bottom: 1]])
+
+private let baseInputClass =
```

Comment:
a lot of this is shareable form styles that I can extract to `PointFreeBaseStyles` soon